static/js/lib/api.js

```diff
@@ -15,8 +17,16 @@ import type {
 } from "../flow/discussionTypes"
 import type { CommentResponse } from "../reducers/comments"
 
-export function getFrontpage(): Promise<Post> {
-  return fetchWithAuthFailure(`/api/v0/frontpage/`)
+const getPaginationQS = R.compose(
+  toQueryString,
+  R.reject(R.isNil),
+  getPaginationParams
+)
+
+export function getFrontpage(params: {
+  [key: string]: ?string
```

Comment:
Can we make a more specific type for this, since we know this should only be `after`, `before`, and `count`?